.eslintrc.js

```diff
@@ -510,6 +510,14 @@ module.exports = {
       // typescript and javascript for front and back end
       files: ['x-pack/legacy/plugins/siem/**/*.{js,ts,tsx}'],
       plugins: ['eslint-plugin-node', 'react'],
+      env: {
+        mocha: true,
+        jest: true,
+      },
```

Comment:
setting these properties to true tells eslint to look out for any globals associated with jest and mocha. Unfortunately Cypress doesn't have this functionality in their eslint plugin so I declared them as globals below.